RELEASE.md

```diff
@@ -0,0 +1,34 @@
+Release type: patch
+
+Adds support for use_pydantic_alias parameter in pydantic model conversion.
+Decides if the graphql field name should use the alias name or not.
+
+```python
+from pydantic import BaseModel, Field
+import strawberry
+
+class UserModel(BaseModel):
+      id: int = Field(..., alias="my_alias_name")
+
+@strawberry.experimental.pydantic.type(
+    UserModel, use_pydantic_alias=False
+)
+class User:
+    id: strawberry.auto
+```
+
+If use_pydantic_alias is `False`, the graphql type User will have the field id
+```
+type User {
+      id: Int!
+}
+```
+
+instead of the field myAliasName.
```

Comment:
```suggestion
With `use_pydantic_alias` set to `True` (the default behaviour) the GraphQL type user will use `myAliasName` for the `id` field coming from the Pydantic models (since the field has a `alias` specified`)
```